tendrl/ceph_integration/objects/rbd/atoms/create/__init__.py

```diff
@@ -13,6 +17,54 @@ def __init__(self, *args, **kwargs):
         super(Create, self).__init__(*args, **kwargs)
 
     def run(self):
+        # Checking if a new  pool has to be created or not rbd creation.
+        if self.parameters['Rbd.create_pool'] is True and \
+                        self.parameters['Rbd.pool_id'] is None:
+            # Creating job for pool creation.
+            if ['Rbd.pool_parameters'] in self.parameters:
```

Comment:
This is actually where I had a confusion. Right now the pool_parameters are not there. If the new pool has to be created then the pool_parameters are required. I was expecting the structure to same as parameters field for create pool job. @anivargi Please give your suggestions on this